core/config/config_test.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"chain/database/pg/pgtest"
+	"chain/database/sinkdb"
+	"chain/database/sinkdb/sinkdbtest"
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"testing"
```

Comment:
If you're using `goimports` to automatically insert imports, you can use `$CHAIN/bin/gochainimports` as a drop-in replacement that will do this grouping automatically.